src/en/batcave/src/eu/kanade/tachiyomi/extension/en/batcave/BatCave.kt

```diff
@@ -180,6 +180,9 @@ class BatCave : HttpSource() {
             thumbnail_url = document.selectFirst("div.page__poster img")?.absUrl("src")
             description = document.selectFirst("div.page__text")?.wholeText()
             author = document.selectFirst(".page__list > li:has(> div:contains(Publisher))")?.ownText()
+            genre = (
+                document.select("div.page__tags a").mapNotNull { it.text().takeIf { it.isNotBlank() } } + listOf("Comic")
+                ).joinToString(", ")
```

Comment:
```suggestion
            genre = buildList {
                document.select("div.page__tags a").forEach { tag ->
                    add(tag.text())
                }
                add("Comic")
            }.joinToString()
```
- It is unlikely that the site will have genre tags with no text, and app does filter out any potential empty genres, so no need to check empty
- `joinToString` defaults to ", "